src/common/Types.ts

```diff
@@ -547,6 +547,8 @@ export interface TextPropsShared extends CommonProps {
     elevation?: number;
 
     onPress?: (e: SyntheticEvent) => void;
+
+    id?: string; // Web only. Needed for accessibility.
```

Comment:
Please add to appropriate documentation (".md") file. Props are listed in alphabetical order in the documentation. Docs also explain the default value. I think the default value in this case is undefined.